include/material/newtonian.h

```diff
@@ -65,10 +65,12 @@ class Newtonian : public Material<Tdim> {
 
   //! Density
   double density_{std::numeric_limits<double>::max()};
-  //! Bulk modulus
+  //! Fluid Bulk modulus
   double bulk_modulus_{std::numeric_limits<double>::max()};
-  //! mu - constant plastic viscosity [N s / m^2 or kg / m / s]
-  double mu_{std::numeric_limits<double>::max()};
+  //! Fluid Dynamic viscosity
+  double dynamic_viscosity_{std::numeric_limits<double>::max()};
+  //! Fluid Incompressibility
+  bool incompressible_{false};
```

Comment:
Since we are almost always dealing with weakly compressible, could we remove this and create a new class `NewtonianFluid` that can deal with returning shear stresses for your projection method without the pressure term.

Reply:
hmmm... That was my initial thought, but after we clarify that Newtonian also takes strain_rate. I dont see differentiating the materials is necessary. I guess, making NewtonianFluid class will cause even more confusion than merely adding a boolean as material property.

I am at the moment developing the projection method, and I am finding problems with stresses, that's why I came up with this. Perhaps, we can talk on Monday? @srhgk2 @kks32

Reply:
All you need is shear stress, i.e. the deviatoric part of your stress, because the volumetric part is handled by the projection method. It would be better to see your implementation to see why you are having problems.

Reply:
I am sure that this is not the only bug. If you add pressure in the `compute_stress` function to the returned `pstress`, that causes an error already right?

Reply:
So, create a new material model as @kks32 suggested.

Reply:
My argument is the same as I wrote, but if you think that's the best way, I will do it. Thanks

Reply:
I can understand that its not good to duplicate every fluid model. But until we come up with a better solution, I would suggest let's go with the most straightforward way.

Reply:
I think this is a good time to rethink about this. Let's discuss it soon.